src/lock_file/update.rs

```diff
@@ -1099,14 +1099,18 @@ fn make_unsupported_pypi_platform_error(
     }
 
     // Find all the features that are excluding the current platform.
-    let features_without_platform = grouped_environment.features().filter_map(|feature| {
-        let platforms = feature.platforms.as_ref()?;
-        if !platforms.value.contains(&current_platform) {
-            Some((feature, platforms))
-        } else {
-            None
-        }
-    });
+    let features_without_platform = grouped_environment
+        .environments()
+        // The default feature is included if its platforms are included in the environment
+        .flat_map(|e| e.features(e.manifest().from_default_feature.platforms))
```

Comment:
Or should this be
```suggestion
        .flat_map(|e| e.features(e.manifest().from_default_feature.pypi_dependencies))
```

Reply:
Sorry, don't really get this, why change from `platforms` to `pypi_dependencies`?

Reply:
That part of the code tries to identifies features that do not include a given platform, as part of generating an error related to pypi-dependencies. Basically, I am not sure when to include or exclude the default feature: if pypi deps from the default feature are excluded from the env, should we include / exclude the default feature here?